source/Introduction/How-SpaceROS-Differs.rst

```diff
@@ -1,22 +1,118 @@
 How Space ROS Differs from ROS 2
 ================================
 
-TODO:
-
-Provide a high-level overview of the various features of Space ROS and describe how Space ROS is different from ROS 2.
-
-Some example topics:
-
-* Space ROS website (and domain names)
-* Space ROS documentation site (this site)
-* Space ROS github organization
-* Additional static analysis tools and SARIF output (IKOS, Cobra, MC/DC testing, etc.)
-* The Space ROS requirements tools and processes
-* Space ROS Quality Level(s)
-* Earthly for local developer builds and CI
-* C++ PMR memory allocator
-* The Eventing & Telemetry Subsystem
-* Space-related demo applications
-* Space-related simulation assets
-* RTEMS
-* cFS/ROS 2 Bridge
+To date, work has focused on establishing the foundation for Space ROS, including builds and infrastructure, code analysis tools, requirements tools and processes, and space-specific functionality.
+These efforts extend the foundation provided by ROS 2.
+The following sections list the differences between Space ROS and the standard ROS 2 distributions.
+
+Builds and Infrastructure
+-------------------------
+..
+  Space ROS builds include a subset of ROS 2 packages.
```

Comment:
Comment directive. Intentional or should it be removed?

Reply:
Intentional, but now removed.